bower.json

```diff
@@ -15,11 +15,12 @@
   ],
   "repository": "https://github.com/Polymer/paper-tabs/tree/0.8-preview",
   "dependencies": {
+    "iron-behaviors": "polymerelements/iron-behaviors#^0.9.0",
```

Comment:
I usually do this after I merge the PR, just because I've noticed sometimes it becomes a race condition for different PRs.